group_vars/matrix-servers

```diff
@@ -222,6 +223,8 @@ matrix_ssl_domains_to_obtain_certificates_for: |
     ([matrix_server_fqn_matrix])
     +
     ([matrix_server_fqn_riot] if matrix_riot_web_enabled else [])
+    +
+    ([matrix_server_fqn_dimension] if matrix_dimension_enabled else [])
   }}
 
 ######################################################################
```

Comment:
Let's add a `matrix-dimension` configuration section to this `group_vars/matrix-servers` file.
It should be added after the `matrix-coturn` section (because the sections are ordered alphabetically).

We can set `matrix_dimension_enabled: false`, because enabling it by default from the get-go seems hard (it requires a working server, a user and a working admin access token).

`false` is the default value, so the above doesn't make any difference, but:
- it explicitly sets it, which is better for getting an overview of how the playbook wires the roles
- we may add other variables to that section in the future as well

Thinking of which, we **do** need to add one more variable from the very beginning (I'm discussing it in another review comment):

```yaml
# Normally, matrix-nginx-proxy is enabled and nginx can reach Dimension over the container network.
# If matrix-nginx-proxy is not enabled, or you otherwise have a need for it, you can expose
# the Dimension HTTP port to the local host (`127.0.0.1:8184`).
matrix_dimension_container_expose_port: {{ not matrix_nginx_proxy_enabled }}
```